openc3/lib/openc3/io/json_api_object.rb

```diff
@@ -77,14 +77,9 @@ def request(*method_params, **keyword_params)
       endpoint = method_params[1]
       @mutex.synchronize do
         kwargs = _generate_kwargs(keyword_params)
-        for attempt in 1..3
-          @log = [nil, nil, nil]
-          connect() if !@http
-          response = _send_request(method: method, endpoint: endpoint, kwargs: kwargs)
-          return response unless response.code >= 500
-          sleep attempt
-        end
-        return nil
+        @log = [nil, nil, nil]
+        connect() if !@http
+        return _send_request(method: method, endpoint: endpoint, kwargs: kwargs)
```

Comment:
Ahh, so this was the source the long delays .. especially with the sleep attempt in there!